recipes/imagemgagick/all/conanfile.py

```diff
@@ -0,0 +1,349 @@
+                              upgrade_project=False,
+                              platforms={'x86': 'Win32', 'x86_64': 'x64'})
+
+        with tools.chdir(os.path.join('VisualMagick', 'coders')):
+            pattern = 'IM_MOD_*_%s.vcxproj' % suffix if self.options.shared else 'CORE_coders_%s.vcxproj' % suffix
+            projects = glob.glob(pattern)
+            for project in projects:
+                msbuild = MSBuild(self)
+                msbuild.build(project_file=project,
+                              upgrade_project=False,
+                              platforms={'x86': 'Win32', 'x86_64': 'x64'})
+
+    def _build_configure(self):
+        with tools.chdir(self._source_subfolder):
+            env_build = AutoToolsBuildEnvironment(self, win_bash=self._is_mingw_windows)
+            args = ['--disable-openmp',
+                    '--disable-docs',
+                    '--with-perl=no',
+                    '--with-x=no'
+                    ]
+
+            os.makedirs('pkgconfig')
+            pkg_config_path = os.path.abspath('pkgconfig')
+            pkg_config_path = tools.unix_path(pkg_config_path) if os.name == 'nt' else pkg_config_path
+
+            for dep in self.deps_cpp_info.deps:
+                self._copy_pkg_config(dep)
+
+            if self.options.shared:
+                args.extend(['--enable-shared=yes', '--enable-static=no'])
+            else:
+                args.extend(['--enable-shared=no', '--enable-static=yes'])
+            args.append('--enable-hdri=yes' if self.options.hdri else '--enable-hdri=no')
+            args.append('--with-quantum-depth=%s' % self.options.quantum_depth)
+            args.append('--with-zlib=yes' if self.options.zlib else '--with-zlib=no')
+            args.append('--with-bzlib=yes' if self.options.bzlib else '--with-bzlib=no')
+            args.append('--with-lzma=yes' if self.options.lzma else '--with-lzma=no')
+
+            args.append('--with-lcms=yes' if self.options.lcms else '--with-lcms=no')
+            args.append('--with-openexr=yes' if self.options.openexr else '--with-openexr=no')
+            args.append('--with-jpeg=yes' if self.options.jpeg else '--with-jpeg=no')
+            args.append('--with-openjp2=yes' if self.options.openjp2 else '--with-openjp2=no')
+            args.append('--with-png=yes' if self.options.png else '--with-png=no')
+            args.append('--with-tiff=yes' if self.options.tiff else '--with-tiff=no')
+            args.append('--with-webp=yes' if self.options.webp else '--with-webp=no')
+            args.append('--with-xml=yes' if self.options.xml else '--with-xml=no')
+            args.append('--with-freetype=yes' if self.options.freetype else '--with-freetype=no')
+            args.append('--with-utilities=yes' if self.options.utilities else '--with-utilities=no')
+            env_build.configure(args=args, pkg_config_paths=[pkg_config_path])
+            env_build.make()
+            env_build.install()
```

Comment:
Please move the `env_build.install()` to the `package` method.
You can cache the env_build variable inside the class to avoid re-running `configure` in `package()` after `build()`.
Check other autotools recipes in this repo (grep for `_configure_autotools`)